build.gradle.kts

```diff
@@ -82,3 +90,26 @@ fun includedBuildTasks(taskName: String, filter: (IncludedBuild) -> Boolean = {
         .filter { it.name != "build-logic" }
         .filter(filter)
         .mapNotNull { it.task(":$taskName") }
+
+
+tasks.wrapper {
+    val gradleVersion = "8.7"
```

Comment:
just to clarify: after PR merged to update the version of Gradle, we need:
1. set `gradleVersion` to the new value
2. go to `https://gradle.org/release-checksums/` and copy the checksum
3. set `distributionSha256Sum` to the new value
4. run `gradle wrapper`

correct?

Previously it was just `gradle wrapper --gradle-version=8.8`

Reply:
Good point, I will double check and document the steps. Normally I update `gradle.properties` but yes, there are multiple ways.

> Previously it was just gradle wrapper --gradle-version=8.8

And then re-run `gradle wrapper`, otherwise the wrapper scripts won't be updated 🙃

Reply:
TIL how to update Gradle using `distributionSha256Sum`. It is not simple :(

Hopefully the instructions are clear - WDYT?